AppCenterDataStorage/AppCenterDataStorage/Internal/LocalStore/MSDBDocumentStore.m

```diff
@@ -98,6 +91,25 @@ - (BOOL)upsertWithToken:(MSTokenResult *)token
   return result == SQLITE_OK;
 }
 
+- (BOOL)upsertWithToken:(MSTokenResult *)token
+        documentWrapper:(MSDocumentWrapper *)documentWrapper
+              operation:(NSString *_Nullable)operation
+       deviceTimeToLive:(NSInteger)deviceTimeToLive {
+  /*
+   * Compute expiration time as now + device time to live (in seconds).
+   * If device time to live is set to infinite, set expiration time as null in the database.
+   * Note: If the cache/store is meant to be disabled, this method should not even be called.
+   */
+
+  // This is the same as [[NSDate date] timeIntervalSince1970] - but saves us from allocating an NSDate.
+  NSTimeInterval now = NSDate.timeIntervalSinceReferenceDate + NSTimeIntervalSince1970;
+  NSTimeInterval expirationTime = -1;
```

Comment:
It would be nice to use `kMSDataStoreTimeToLiveInfinite` constant instead of a "magic number" like `-1`, especially the constant is already used in line 107.

Reply:
Addressed in #1532.